src/main/kotlin/FinalProject/Main.kt

```diff
@@ -1,5 +1,93 @@
 package FinalProject
 
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvWriter
+import com.univocity.parsers.csv.CsvWriterSettings
+
+/* the scenario function creates a dozen cars, puts them into a road network,
+ * simulates the traffic and prints the jammed / non-jammed for every car.
+ */
+fun scenario(Cars: MutableList<Car>) {
+	/*val car1 = Car(true)
+	val car2 = Car(false)
+	val car3 = Car(true)
+	val car4 = Car(true)
+	val car5 = Car(false)
+	val car6 = Car(true)
+	val car7 = Car(true)
+	val car8 = Car(false)
+	val car9 = Car(true)
+	val car10 = Car(true)
+	val car11 = Car(false)
+	val car12 = Car(true) */
+	
+	//val Cars = listOf(car1, car2, car3, car4, car5, car6, car7, car8, car8, car9, car10, car11, car12)
+	val rn = RoadNetwork(7, Cars)
+	rn.simulationStep()
+	rn.printCarsJammed()
+}
+
 fun main(args: Array<String>) {
-    println("Hello, World!")
+	val Cars : MutableList<Car> = parseCSV()
+
+	scenario(Cars)

```

Comment:
The `scenario` function is meant to execute an entire simulation (i.e. create cars, simulate, write results) without any CSV input/output. The reason is that you should always have one way to run the simulation _without_ having to rely on CSV parsing (which is easy to break accidentally).

So please change `scenario` to create its own cars and write a second function (maybe `csvScenario`) that loads from CSV, simulates, and writes to CSV. `main` should then execute one or both of these functions.